cypress/e2e/pages/homePage/index.cy.js

```diff
@@ -17,37 +53,109 @@ const testSuites = [
     path: '/dari',
     runforEnv: ['local', 'test'],
     service: 'dari',
-    tests,
+    pageIdentifier: 'dari.page',
+    siteId: 142,
+    applicationType: 'responsive',
+    contentType: 'index-home',
+    useReverb: true,
+    tests: [
+      ...tests,
+      ...atiAnalyticsNavigationComponentTestsAssertions,
+      assertMessageBannerComponentView,
+      assertMessageBannerComponentClick,
+    ],
   },
   {
     path: '/kyrgyz',
-    runforEnv: ['local', 'test', 'live'],
+    runforEnv: ['local', 'live'],
     service: 'kyrgyz',
-    tests,
+    pageIdentifier: 'kyrgyz.page',
+    siteId: 58,
+    applicationType: 'responsive',
+    contentType: 'index-home',
+    useReverb: true,
+    tests: [
+      ...tests,
+      ...atiAnalyticsNavigationComponentTestsAssertions,
+      assertMessageBannerComponentView,
+      assertMessageBannerComponentClick,
+      assertMostReadComponentView,
+      assertMostReadComponentClick,
+    ],
   },
   {
     path: '/magyarul',
     runforEnv: ['local', 'test'],
     service: 'magyarul',
-    tests,
+    pageIdentifier: 'magyarul.page',
+    siteId: 30,
+    applicationType: 'responsive',
+    contentType: 'index-home',
+    useReverb: true,
+    tests: [...tests, assertPageView],
```

Comment:
Let's add `assertPageView` to the `tests` array on line 35 so that every test runs it.